xml/deployment_hwrecommend.xml

```diff
@@ -178,18 +188,10 @@
  [...]</screen>
     <para>
      Additionally, if you need to specify a per-monitor public address or
-     network, you need to add a
-     <literal>[mon.<replaceable>X</replaceable>]</literal> section for each
-     monitor:
+     network, adjust the <filename>ceph.conf</filename> file accordingly:
     </para>
-<screen>[mon.MON1]
- public network = 192.168.123.0/24
-
- [mon.MON2]
- public network = 1.2.3.0/24
-
- [mon.MON3]
- public network = 242.12.33.12/0</screen>
+<screen>[global]
+ public network = 192.168.123.0/24, 1.2.3.0/24, 242.12.33.12/0</screen>
```

Comment:
Question.
The 1st example is in the public Class C address range (192.168.0.1–192.168.254.254, from memory). 

If we are using these as examples, then should the others **also** be public? 10.x.y.z & 176.16.y.z?

If not & we want real ranges... SUSE's ranges?

1.2.3.0 is a real address range and it's not ours. 242.x.y.z I don't know.

Reply:
I think @asettle is away for a while now, so @smithfarm I would really appreciate your opinion on this.